config/environments/test.rb

```diff
@@ -1,4 +1,7 @@
 require "active_support/core_ext/integer/time"
+if Rails.env.test?
```

Comment:
Same here. Conditional is no needed since file is loaded only in the test environment.